fronts-client/src/components/App.tsx

```diff
@@ -139,28 +141,44 @@ const BackgroundHeader = styled.div`
 	width: 100%;
 `;
 
-const App = () => (
-	<ThemeProvider theme={styleTheme}>
-		<DropDisabler>
-			<BannerNotification />
-			<AppContainer>
-				<BackgroundHeader>
-					<SectionHeaderWithLogo greyHeader={true} />
-				</BackgroundHeader>
-				<Switch>
-					<Route {...frontsEditPathProps} component={FrontsEdit} />
-					<Route {...issuePathProps} component={FrontsEdit} />
-					<Route {...frontsFeatureProps} component={FeaturesView} />
-					<Route exact path="/" component={Home} />
-					<Route exact path={manageEditions} component={ManageView} />
-					<Route component={NotFound} />
-				</Switch>
-			</AppContainer>
-			<PlaceholderAnimation />
-			<AppFonts />
-			<OptionsModal />
-		</DropDisabler>
-	</ThemeProvider>
-);
+function usePixelTracking() {
+	const location = useLocation();
+	React.useEffect(() => {
+		if (pageConfig.telemetryUrl) {
+			void fetch(`${pageConfig.telemetryUrl}&path=${location.pathname}`, {
```

Comment:
It looks like `location.pathname` return the relative path, meaning in the case we are dropping the `/v2`:

<img width="622" alt="image" src="https://github.com/user-attachments/assets/7e4d1dea-b080-4d4c-96c7-b3e6e4e84d55" />

I expect if we were to call `window.location.pathname` here it would work as we would hope.

Reply:
Yeah, `window.location.pathname` works for us.  You are right that we need to use `useLocation` hook here.